guides/developer/service-overviews/orca.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: single
+title:  "Orca Internals Overview"
+sidebar:
+  nav: guides
+---
+
+Spinnaker is known and often loved for enabling accessible, highly configurable and composable workflows through its pipelines and tasks. This is largely thanks to the code that makes up Orca.
+
+From Orca's [README](https://github.com/spinnaker/orca/blob/master/README.md):
+
+> Orca is the orchestration engine for Spinnaker. It is responsible for taking a pipeline or task definition and managing the stages and tasks, coordinating the other Spinnaker services.
+
+Orca is a stateless, horizontally scalable service. Spinnaker's execution state is persisted to a backend store and work is distributed evenly through a work queue. So long as your backend persistence layer has capacity, you can freely add or remove Orca servers to match workload demands.
+
+# Internals
+
+## Domain
+
+The primary domain model is an **Execution**, of which there are two types: `PIPELINE` and `ORCHESTRATION`. The `PIPELINE` type is for pipelines while `ORCHESTRATION` is unscheduled, ad-hoc API-submitted actions and what you see in the "Tasks" tab in an Application view. At this point in Spinnaker’s life, these two types are nearly programmatically identical, the key difference being that a Pipeline has a predefined persistent configuration, whereas an Orchestration is arbitarily configured at request-time and serial in execution. 
+
+For one Execution, there are one or many **Stages** organized as a DAG (Directed Acyclic Graph). An Execution supports concurrent branching logic of Stages. These Stages define higher-order, user-friendly actions and concepts, such as "Resize Server Group" or "Deploy" and can be chained together to form complex delivery workflows.
+
+Within a Stage, there are one or more Tasks. A **Task** can be considered an atomic unit of work, focused on a single action. If you are familiar with the Spinnaker UI, you'll note that for every Stage, there are multiple line items that correlate to a Stage's runtime: These are tasks. Tasks are always executed sequentially and serially.
```

Comment:
Might reemphasize that these tasks are NOT what's in the UI's Task tab. Also there are two spots where "multiple line items" show up in the execution UI, which might be confusing (the UI has a notion of also stage "steps")